base_tier_validation/models/tier_validation.py

```diff
@@ -166,25 +181,46 @@ def _notify_accepted_reviews(self):
     def _notify_accepted_reviews_body(self):
         return _('A review was accepted')
 
+    def _add_comment(self, validate_reject):
+        wizard = self.env.ref(
+            'base_tier_validation.view_comment_wizard')
+        definition_ids = self.env['tier.definition'].search([
+            ('model', '=', self._name),
+            '|', ('reviewer_id', '=', self.env.user.id),
+                 ('reviewer_group_id', 'in',
+                  self.env.user.groups_id.ids)
+        ])
+        return {
+            'name': _('Comment'),
+            'type': 'ir.actions.act_window',
+            'view_type': 'form',
+            'view_mode': 'form',
+            'res_model': 'comment.wizard',
+            'views': [(wizard.id, 'form')],
+            'view_id': wizard.id,
+            'target': 'new',
+            'context': {
+                'default_view_id': self.id,
```

Comment:
self.id is the view_id? I though it is tier.validation object.

Reply:
Edited